tests/x11regressions/firefox/firefox_extcontent.pm

```diff
@@ -19,7 +19,7 @@ sub run {
     my ($self) = @_;
     $self->start_firefox;
 
-    my $ext_link = "http://mirror.bej.suse.com/dist/install/SLP/SLE-12-Server-GM/x86_64/dvd1/";
+    my $ext_link = "http://mirror.bej.suse.com/dist/install/SLP/SLE-12-SP3-Server-GM/x86_64/dvd1/";
```

Comment:
What's about using VERSION variable in here? Or moving this variable to the test suite and just reading value here?

Reply:
what is on that server is unrelated to the tested version. We're actually not testing that server, but that firefox can open random urls.

Reply:
I just thought if we can "predict" url instead of changing it every time it's outdated. Last time it was 2 years ago, so should be fine ;)